### PR TITLE
Hide "Ravi's recommendations" widget title if there are no recommendations

### PR DESCRIPTION
**Pull request by ilicfilip**
Fixes https://github.com/Emilia-Capital/progress-planner/issues/123

### CODE REVIEW

**Comment by aristath** [comment]
Right now it's impossible to hide all tasks... There's always at least one - and I think that's a bug.
If I try to snooze all tasks, after refreshing the page I always see a new task to create a long post.
@ilicfilip Can you replicate? 🤔

**Comment by ilicfilip** [comment]
When I snooze the local suggested task I get an empty list and it stays e
Screenshot 2024-11-18 at 10 26 32
mpty even when I refresh the page
(that's why I opened this issue).

File: views/page-widgets/suggested-tasks.php [hunk 1]
```diff
@@ -31,9 +31,10 @@
 
 <hr>
 
-<h2 class="prpl-widget-title">
-	<?php \esc_html_e( 'Ravi\'s recommendations', 'progress-planner' ); ?>
-</h2>
+<div class="prpl-dashboard-widget-suggested-tasks">
+	<h2 class="prpl-widget-title">
+		<?php \esc_html_e( 'Ravi\'s recommendations', 'progress-planner' ); ?>
+	</h2>
```
Comment (aristath) [inline views/page-widgets/suggested-tasks.php:37]: @ilicfilip Was this change done by mistake?

Comment (ilicfilip) [reply views/page-widgets/suggested-tasks.php:37]: I think I added wrapper so I can target that h2 specifically, but closing </div> was removed by some merge probably.

Comment (ilicfilip) [reply views/page-widgets/suggested-tasks.php:37]: Yeah, it was removed here: https://github.com/Emilia-Capital/progress-planner/pull/125/commits/167bfba14be6bf561294b994c32064447f3b078d

Comment (aristath) [reply views/page-widgets/suggested-tasks.php:37]: 🤔 Why can't we use `.prpl-widget-wrapper.prpl-suggested-tasks` as a selector?

**Pull request merged**